sys/riscv/cheri/cheri_coaccept.S

```diff
@@ -152,7 +152,7 @@ _C_LABEL(switcher_coaccept):
 	 */
 	sub	t3, a4, t5
 	bgez	t3, 2f
-	addi	t5, a4, 0
+	move	t5, a4
```

Comment:
mv is the canonical name for it (didn't even know move was accepted as an alias)